traffic-id.yaml

```diff
@@ -126,6 +157,84 @@ tls-sni-patterns:
       - instagram.com
       - cdninstagram.com
 
+  - id: feedly
+    labels:
+      - news
+      - rss
+    patterns:
+      - feedly.com
+
+  - id: nos
```

Comment:
Is there a longer ID that can be used? My feeling is that this takes up a short name that could potentially be used for other IDs in the future. Not sure what.

Reply:
perhaps a country namespace? e.g. nl/nos or NL-nos or something along those lines. Guess the forward slash might confuse the flowbits that are set.

Reply:
Namespace is a good idea. Not sure if a / would be an issue for not. The logger only looks for a prefix of "traffic/id/" then takes the rest as the ID. So "nl/nos" should be logged as:
```
{
  "traffic": {
    "id": ["nl/nos"]
  }
}

which I think could be a good option.